opm/simulators/linalg/PropertyTree.cpp

```diff
@@ -121,4 +141,8 @@ template int PropertyTree::get(const std::string& key, const int& defValue) cons
 template std::size_t PropertyTree::get(const std::string& key, const std::size_t& defValue) const;
 template bool PropertyTree::get(const std::string& key, const bool& defValue) const;
 
+template std::optional<std::vector<int>>
+PropertyTree::get_child_items_as_vector<int>(const std::string& child) const;
+template std::optional<std::vector<double>>
+PropertyTree::get_child_items_as_vector<double>(const std::string& child) const;
```

Comment:
newline

Reply:
> newline

Would you mind adding a little more context, please?  What aspect of newline would like to see changed here?

Reply:
missing  new line before the ending brace.